src/app/services/mocks/constraint.service.mock.ts

```diff
@@ -63,4 +66,32 @@ export class ConstraintServiceMock {
   public identifyDraggedElement(): Concept {
     return new Concept();
   }
+
+  checkAllVariables(b: boolean) {
```

Comment:
`checkAllVariables` is the wrong name, should be something like `setVariableSelection`.

Reply:
will change